kubernetes_state/tests/fixtures/prometheus.txt

```diff
@@ -70,7 +70,7 @@ kube_deployment_created{deployment="tiller-deploy",namespace="kube-system"} 1.50
 # HELP kube_deployment_labels Kubernetes labels converted to Prometheus labels.
 # TYPE kube_deployment_labels gauge
 kube_deployment_labels{deployment="failingtest",label_app="failingtest",namespace="default"} 1
-kube_deployment_labels{deployment="tiller-deploy",label_app="helm",label_name="tiller",namespace="kube-system"} 1
+kube_deployment_labels{deployment="tiller-deploy",label_tags_datadoghq_com_env="dev",label_tags_datadoghq_com_service="tiller",label_tags_datadoghq_com_version="xyz",label_app="helm",label_name="tiller",namespace="kube-system"} 1
```

Comment:
thanks for the tests, could you please cover all the targeted objects in this PR not only deployments and pods?

Reply:
Yes, will add those!